pilot/pkg/bootstrap/server.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"code.cloudfoundry.org/copilot"
 	"github.com/davecgh/go-spew/spew"
+	"github.com/emicklei/go-restful"
```

Comment:
did you check and see if this dependency is used anywhere else? May be able to remove it from vendor?